simulation/tests/test_agents.py

```diff
@@ -737,3 +737,93 @@ def test_household_ability_to_choose_heat_pump_as_option_depends_on_model_heat_p
         )
 
         assert all(heat_pump in heating_system_options for heat_pump in HEAT_PUMPS)
+
```

Comment:
I think because these are testing how the agents behave under a boiler ban, WDYT about moving them all into their own class:
```
class TestAgentsWithBoilerBan:
```

It might help us segment the desired behaviour under the boiler ban from general behaviour?